### PR TITLE
Update XRPC procedures to use request body by default

### PR DESCRIPTION
**Pull request by pfrazee**
- Updates code generated by lex-cli to expect the first param to be the input body, and moves query params to the `qp` field in the `opts`.
- Updates procedure lexicons to favor input bodies.
- Updates all code to use new lexicons and API signatures.

### CODE REVIEW

File: lexicons/bsky.app/updateProfile.json [hunk 1]
```diff
@@ -3,7 +3,6 @@
   "id": "app.bsky.updateProfile",
```
Comment (dholms) [inline lexicons/bsky.app/updateProfile.json:3]: think you missed `postNotificationsSeen` in the bsky lexicon

File: packages/api/tests/session.test.ts [hunk 2]
```diff
@@ -27,10 +27,11 @@ describe('session', () => {
     const sessions: (Session | undefined)[] = []
     client.sessionManager.on('session', (session) => sessions.push(session))
 
-    const { data: account } = await client.com.atproto.createAccount(
-      {},
-      { username: 'alice.test', email: 'alice@test.com', password: 'password' },
-    )
+    const { data: account } = await client.com.atproto.createAccount({
+      username: 'alice.test',
```
Comment (dholms) [inline packages/api/tests/session.test.ts:31]: so much better

File: lexicons/atproto.com/repoCreateRecord.json [hunk 3]
```diff
@@ -3,15 +3,31 @@
...
+        },
+        "collection": {
+          "type": "string",
+          "description": "The NSID of the record collection."
+        },
+        "validate": {
+          "type": "boolean",
+          "default": true,
+          "description": "Validate the record?"
+        },
```
Comment (devinivy) [inline lexicons/atproto.com/repoCreateRecord.json:24]: Just a note to self (and @dholms I suppose!) to look into xrpc server, ensure it's applying these defaults during validation.  I am pretty sure I witnessed validation not occurring by default when creating a post via curl.

File: packages/api/tests/session.test.ts (hunk 2 above)
Comment (pfrazee) [reply packages/api/tests/session.test.ts:31]: lol ya

File: packages/api/src/client/index.ts [hunk 4]
```diff
@@ -98,6 +98,11 @@ export * as AppBskyProfile from './types/app/bsky/profile'
 export * as AppBskyRepost from './types/app/bsky/repost'
 export * as AppBskyUpdateProfile from './types/app/bsky/updateProfile'
 
+export const APP_BSKY = {
+  ActorScene: 'app.bsky.actorScene',
+  ActorUser: 'app.bsky.actorUser',
+}
```
Comment (devinivy) [inline packages/api/src/client/index.ts:104]: _Yes_

File: lexicons/atproto.com/repoCreateRecord.json (hunk 3 above)
Comment (pfrazee) [reply lexicons/atproto.com/repoCreateRecord.json:24]: Actually you're totally right -- the json schema defaults are _not_ getting applied. I had to modify the repoCreateRecord endpoint to do that manually

File: packages/api/src/client/index.ts (hunk 4 above)
Comment (pfrazee) [reply packages/api/src/client/index.ts:104]: The tiger
He destroyed his cage
Yes
YES
The tiger is out

File: lexicons/bsky.app/updateProfile.json (hunk 1 above)
Comment (pfrazee) [reply lexicons/bsky.app/updateProfile.json:3]: fixed

File: lexicons/atproto.com/repoCreateRecord.json (hunk 3 above)
Comment (dholms) [reply lexicons/atproto.com/repoCreateRecord.json:24]: 👍 agreed
general note: let's make issues for stuff like this when we come across it

File: packages/api/src/client/index.ts (hunk 4 above)
Comment (dholms) [reply packages/api/src/client/index.ts:104]: hahaha incredible

**Review by dholms — APPROVED** [review]
beautiful 🙌

**Pull request merged**